using-preview-to-edit-images.md

```diff
@@ -0,0 +1,90 @@
+
+Double click your chosen image to open it in Preview ![Logo for Preview software](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-05a.png "Logo for Preview software").<br>
+⌘ A (Select all)<br>
+⌘ C (Copy)
+
+Click into the template<br>
+⌘ V (Paste)
+
+Now your image is inside the template in Preview ![Logo for Preview software](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-05a.png "Logo for Preview software"). The image is probably too big 🎪 for the template so we need to shrink it and position it.
+
+Left-click and pull ![Hand icon to illustrate pulling](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-00a.png "Hand icon to illustrate pulling") the image within the template until you can see the blue dot on the top right corner.<br>
+![Template image being edited](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-12.jpg "Template image being edited")
+
+Left-click and drag ![Diagonal arrow icon to illustrate dragging](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-11a.png "Diagonal arrow icon to illustrate dragging") the blue dot on the corner to the bottom left. This will shrink your image.<br>
+![Template image being edited](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-01.jpg "Template image being edited")
+
+Then left-click on the actual image and pull ![Hand icon to illustrate pulling](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-00a.png "Hand icon to illustrate pulling") it back into the visible part of the template.<br>
+![Template image being edited](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-15.jpg "Template image being edited")<br><br>
+![Template image being edited](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-16.jpg "Template image being edited")
+
+Repeat ![Circular arrows repeat icon](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-17a.png "Circular arrows repeat icon") : Left-click and drag ![Diagonal arrow icon to illustrate dragging](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-11a.png "Diagonal arrow icon to illustrate dragging") the blue dot on the corner to the bottom left. This will shrink your image.
+Then left-click on the actual image and pull ![Hand icon to illustrate pulling](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-00a.png "Hand icon to illustrate pulling") it back into the visible part of the template.
+
+Until your image is the desired size. 
+
+Now position ![Hand icon to illustrate pulling](https://github.com/redbadger/websites-guide/blob/images/assets/edit-image-00a.png "Hand icon to illustrate pulling") your image to your chosen composition.<br>
```

Comment:
It's a very good document. I followed these steps to resize a sponsor's image with Becca. We got a bit stuck at this stage. What happens if the image is a different aspect ratio to the template. Do you just position it so that it fits inside the template and not worry if the template is bigger?